FinTech.Logic/BusinessLogic/Authentication/AuthenticationBL.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FinTech.Data;
+using System.Linq;
+using FinTech.Common;
+
+namespace FinTech.Logic
+{
+    public class AuthenticationBL
+    {
+        public ApiResponse<bool> Register(UserDTO userDto)
+        {
+            var apiResponse = new ApiResponse<bool>();
+
+            if (!InputValidationHelper.IsValidName(userDto.FirstName))
+            {
+                apiResponse.IsSuccess = false;
```

Comment:
Is apiResponse.IsSuccess = false.

Is this really required?
What was default value for bool?